docs/sources/slack.md

```diff
@@ -1,93 +1,118 @@
-# Slack Source for Knative
+# Event source for Slack
 
-Slack Source enables integration between slack messages read by a bot users and Knative Eventing.
+This event source uses the [Slack Events API][slack-events-api] through a [bot user][slack-bot-user] to ingest events into Triggermesh.
```

Comment:
are those anchors or links ? I don't think they render

Reply:
nice catch!
looks like I deleted the links at some copy/paste

fixed at https://github.com/triggermesh/docs/pull/26/commits/b5efa90629bada71a59180cf29402e5d88db1416